### PR TITLE
[5.2] Add array_trim helper

### PR DESCRIPTION
**Pull request by let-aurn**
```
    $users = array(
            array('id' => 1, 'name' => 'John'),
            array('id' => 2, 'name' => 'Jane')
    );
    $fruits = ['banana', 'orange', 'apple'];

    array_trim($users, 'name');    // "John, Jane"
    array_trim($fruits);                // "banana, orange, apple"
```

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:-1: from me. This should just be defined in your app IMO.

File: src/Illuminate/Support/helpers.php [hunk 1]
```diff
@@ -784,3 +784,25 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('array_trim')) {
+    /**
+     * Return the comma separated representation of an array.
+     *
+     * @param  string  $array
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/helpers.php]: incorrect

File: src/Illuminate/Support/helpers.php [hunk 2]
```diff
@@ -784,3 +784,25 @@ function with($object)
...
+     * @param  string  $sep
+     * @param  string  $property
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/helpers.php]: incorrect

**Comment by let-aurn** [comment]
@GrahamCampbell I thought it would be nice to have this helper since I used it a lot. Which criteria should be respected when suggesting helpers?

**Comment by taylorotwell** [comment]
Almost all helpers are actually used by the core framework.

**Comment by vlakoff** [comment]
* [`trim()`](http://php.net/manual/en/function.trim.php) strips characters individually, i.e.:

`trim('-/-foobar/', '/-')` gives `foobar` as it removes `/` and `-`, not the string `/-`.
* You may simply do in your application:
```
implode(', ', array_column($users, 'name'));
implode(', ', $fruits);
```

**Pull request closed without merging**